scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -43,7 +43,7 @@
 	producerURL	URL	Producer URL points to the producer's web presence, if appropriate. Enter an absolute URL where the producer's web site is found, such as http://www.my.org.  	Enter full URL, starting with http://	url	39	<a href="#VALUE" target="_blank" rel="noopener">#VALUE</a>	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	producer	citation
 	producerLogoURL	Logo URL	URL for the producer's logo, which points to this  producer's web-accessible logo image. Enter an absolute URL where the producer's logo image is found, such as http://www.my.org/images/logo.gif.	Enter full URL for image, starting with http://	url	40	<img src="#VALUE" alt="#NAME" class="metadata-logo"/><br/>	FALSE	FALSE	FALSE	FALSE	FALSE	FALSE	producer	citation
 	productionDate	Production Date	Date when the data collection or other materials were produced (not distributed, published or archived).	YYYY-MM-DD	date	41		TRUE	FALSE	FALSE	TRUE	FALSE	FALSE		citation
-	productionPlace	Production Place	The location where the data collection and any other related materials were produced.		text	42		FALSE	FALSE	FALSE	FALSE	FALSE	FALSE		citation
+	productionPlace	Production Place	The location where the data collection and any other related materials were produced.		text	42		FALSE	FALSE	TRUE	TRUE	FALSE	FALSE		citation
```

Comment:
Can you please back out the change to this file?

Reply:
I'm not quite sure what I'm supposed to do. I think this change is intended from my side, but it seems I forgot to create an issue and a PR for this? Do you want me just to undo the change in this current PR?

Reply:
Hey @philippconzett - this change was proposed in #7810 but it has not been merged yet. Can you back out the change for this PR so that we can get this merged, and then work on Production place as part of #7810?

Reply:
Thanks for clarifying, @djbrooke. I have backed out the change and will rather proceed it as part of #7810.

Reply:
I backed out this change in https://github.com/IQSS/dataverse/pull/8024/commits/f51d71eb092d59ec9adc6693454c23d8c7b5b8de